modules/generating-icsp-object-scoped-to-a-registry.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc:
+
+[id="generating-icsp-object-scoped-to-a-registry_{context}"]
+= Widening the scope of the mirror image catalog to reduce the frequency of cluster node reboots
+
+You can scope the mirrored image catalog at the repository level or the wider registry level. A widely scoped `ImageContentSourcePolicy` resource reduces the number of times the nodes need to reboot in response to changes to the resource.
+
+To widen the scope of the mirror image catalog in the `ImageContentSourcePolicy` resource, do the following:
+
+.Prerequisites
+
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+* Configure a mirrored image catalog repository for use in your disconnected cluster.
+
+.Procedure
+
+. Run the following command, specifying values for `<local_registery>` and `<pull_secret_file>`:
++
+[source,terminal]
+----
+$ oc adm catalog mirror <local_registery>/olm/redhat-operators:v1 <local_registery> -a <pull_secret_file> --icsp-scope=registry
+----
++
+where:
+
+<local_registery>:: is the local registry you have configured for your disconnected cluster, for example `local.registry:5000`.
```

Comment:
If the typo in `local_registery` is intentional, +100 for consistency.  I see it in three places in case it was not intentional.

Sorry for missing this in the first review.

Reply:
typo! fixed.